database/bootstrap.go

```diff
@@ -43,7 +47,11 @@ func BootstrapDqlite(ctx context.Context, opt bootstrapOptFactory, logger Logger
 		return errors.Trace(err)
 	}
 
-	dqlite, err := app.New(dir, opt.WithLoopbackAddressOption(), opt.WithLogFuncOption())
+	dqlite, err := app.New(dir,
+		opt.WithLoopbackAddressOption(),
+		opt.WithLogFuncOption(),
+		opt.WithTracingOption(),
```

Comment:
Not sure I'd bother with it at bootstrap.

Reply:
I wanted to keep it symmetrical, therefore we're passing the same things through. I can be persuaded otherwise.